sdks/python/apache_beam/runners/dataflow/internal/dependency.py

```diff
@@ -284,13 +285,15 @@ def stage_job_resources(
     raise RuntimeError(
         'The --temp_location option must be specified.')
```

Comment:
This is just a sanity check as https://github.com/apache/beam/blob/master/sdks/python/apache_beam/runners/dataflow/internal/apiclient.py#L325 would have already assigned temp_location to staging_location had it not been specified. I think we can leave it as is right now.